avalanche-network-runner-sdk/scripts/static-analysis.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -xue
+
+if ! [[ "$0" =~ scripts/static-analysis.sh ]]; then
+  echo "must be run from repository root"
+  exit 255
+fi
+
+# git submodule add https://github.com/googleapis/googleapis
+git submodule update --init --remote
+
+# https://rust-lang.github.io/rustup/installation/index.html
+# rustup toolchain install nightly --allow-downgrade --profile minimal --component clippy
+#
+# https://github.com/rust-lang/rustfmt
+# rustup component add rustfmt
+# rustup component add rustfmt --toolchain nightly
+# rustup component add clippy
+# rustup component add clippy --toolchain nightly
+
+rustup default stable
+cargo fmt --all --verbose -- --check
+
+# TODO: enable nightly fmt
+rustup default nightly
+cargo +nightly fmt --all -- --config-path .rustfmt.nightly.toml --verbose --check || true
+
+# TODO: enable this
```

Comment:
is this not enabled?

Reply:
I meant enable it by removing the `|| true`. Right now, it's only used for warnings but not failing the CI, since it's a nightly build.